src/test/java/teammates/test/cases/webapi/BaseActionTest.java

```diff
@@ -177,6 +180,23 @@ protected void loginAsStudentInstructor(String userId) {
         assertFalse(user.isAdmin);
     }
 
+    protected void updateInstructorWithOnlySectionPrivilege(InstructorAttributes instructor,
```

Comment:
As the we are setting the privilege to `true`, it is better to name the method as `grantPrivilege` instead of `updatePrivilge`.